src/main/java/GeneralService/Main.java

```diff
@@ -2,10 +2,16 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
+@ComponentScan(basePackages = "onlineservices")
 public class Main {
     public static void main(String[] args) {
         SpringApplication.run(Main.class, args);
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext();
```

Comment:
lines 13 14 15 are pointless, spring boot knows about the config file already, remove the lines of code